From 028c81cc5a8a5f9803a91fcf728074194de9b35c Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Wed, 1 Apr 2015 15:55:50 +0800
Subject: [PATCH 0311/1221] MLK-10530-3 ARM: dts: imx7d-sdb: add sii902x label

it's useful to enable/disable it by just using the label.

Signed-off-by: Robby Cai <r63905@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb-hdmi.dts |    6 ++----
 arch/arm/boot/dts/imx7d-sdb.dts      |    2 +-
 2 files changed, 3 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb-hdmi.dts b/arch/arm/boot/dts/imx7d-sdb-hdmi.dts
index 7b4aa27..ac8dc38 100644
--- a/arch/arm/boot/dts/imx7d-sdb-hdmi.dts
+++ b/arch/arm/boot/dts/imx7d-sdb-hdmi.dts
@@ -8,9 +8,7 @@
 
 #include "imx7d-sdb.dts"
 
-&i2c3 {
-	sii902x@39 {
-		status = "okay";
-	};
+&sii902x {
+	status = "okay";
 };
 
diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index cd6662f..b68aa2d 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -364,7 +364,7 @@
 		};
 	};
 
-	sii902x@39 {
+	sii902x: sii902x@39 {
 		compatible = "SiI,sii902x";
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_sii902x>;
-- 
1.7.5.4

